Test "date should set the data" fails when tz not in AEST

Noticed that the tests fail on systems not in AEST(?). Not entirely sure what's going on, but getting things like

* `AssertionError: expected 2014-06-13 00:00:00.000 -0000 to equal 2014-06-12 14:00:00.000 -0000`
* `AssertionError: expected 2014-06-13 00:00:00.000 +0900 to equal 2014-06-12 23:00:00.000 +0900`

I can reproduce this when changing my timezone on my local system, as well as on other systems in other timezones. The issue goes away when I change back.

--- a/test/date.js
+++ b/test/date.js
@@ -16,15 +16,16 @@ describe( 'date', function () {
 			dob: '13 Jun 2014'
 		} );
 
-		personMoldy.dob.should.eql( new Date( '2014-06-12T14:00:00.000Z' ) );
+		is.a.date( personMoldy.dob ).should.be.true;
+		personMoldy.dob.should.eql( new Date( '13 Jun 2014' ) );
 
 		personMoldy.dob = '10 June 2014';
 
 		is.a.date( personMoldy.dob ).should.be.true;
-		personMoldy.dob.should.eql( new Date( '2014-06-09T14:00:00.000Z' ) );
+		personMoldy.dob.should.eql( new Date( '10 June 2014' ) );
 
 		personMoldy.$json().should.eql( {
-			dob: '2014-06-09T14:00:00.000Z'
+			dob: (new Date('10 June 2014')).toISOString()
 		} );
 
 	} );

